theolive/platform/Security/token-based-security.mdx

```diff
@@ -1,6 +1,6 @@
 # Token based security
 
-THEOlive offers the option to enable JWT token security on channel alias level. This can be interesting if you only want valid users to access your stream.
+Dolby OptiView Live offers the option to enable JWT token security on channel alias level. This can be interesting if you only want valid users to access your stream.
```

Comment:
We have not changed the names yet in the docs, so I would probably not change it here.